src/UserPullPieChart.tsx

```diff
@@ -0,0 +1,89 @@
+import { Paper } from '@mui/material';
+import {
+  Cell,
+  Legend,
+  Pie,
+  PieChart,
+  ResponsiveContainer,
+  Tooltip,
+} from 'recharts';
+import { PullRequestKeyMetrics } from './types';
+
+const COLORS = [
+  '#e60049',
+  '#0bb4ff',
+  '#50e991',
+  '#e6d800',
+  '#9b19f5',
+  '#ffa300',
+  '#dc0ab4',
+  '#b3d4ff',
+  '#00bfa0',
+];
+type Props = {
+  authoredPullRequests: PullRequestKeyMetrics[];
+};
+
+function getPullPieData(pullRequests: PullRequestKeyMetrics[]) {
+  const pullAuthorCounter: any = {};
+  let colorCounter = 0;
+  pullRequests.forEach((pull) => {
+    if (pullAuthorCounter[pull.author]) {
+      pullAuthorCounter[pull.author].value =
+        pullAuthorCounter[pull.author].value + 1;
+      pullAuthorCounter[pull.author].totalCodeChanges =
+        pullAuthorCounter[pull.author].totalCodeChanges + pull.totalCodeChanges;
+    } else {
+      pullAuthorCounter[pull.author] = {
+        value: 1,
+        totalCodeChanges: pull.totalCodeChanges,
+        color: COLORS[colorCounter],
+      };
+      colorCounter++;
```

Comment:
What happens if there are more than 9 people on a team?

Reply:
Interestingly it doesn't crash...

![Screen Shot 2022-05-11 at 10 13 35 AM](https://user-images.githubusercontent.com/7210022/167870988-27ddfd3d-1a85-4141-935c-0555094cfdb3.png)

Reply:
Added a check so it will now start reusing the colors.